index.html

```diff
@@ -21,174 +28,217 @@
-    <section class="about">
-      <div>
+    <section id="about">
+      <div id="about_right">
+        <h2 class="about_title">About</h2>
         <p class="par">
           Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod.
         </p>
-        <button class="cv" type="button">Get my CV</button>
-      </div>
-      <div class="languages">
-        <h2 class="about_subtitle">Languages</h2>
-        <ul>
-          <li><h3>JavaScript(ES5/ES6)</h3></li>
-          <li><h3>Ruby</h3></li>
-          <li><h3>HTML</h3></li>
-          <li><h3>CSS</h3></li>
-        </ul>
-        <img class="about_icon" src="/icons/2.svg" alt="hashtag">
+        <button class="cv btn" type="button">Get my CV</button>
       </div>
-      <div class="frameworks">
-        <h2 class="about_subtitle">Frameworks</h2>
-        <ul>
-          <li><h3>Bootstrap</h3></li>
-          <li><h3>Ruby on rails</h3></li>
-          <li><h3>RSpec</h3></li>
-          <li><h3>Capibara</h3></li>
-          <li><h3>Selenium</h3></li>
-        </ul>
-        <img class="about_icon" src="/icons/square.svg" alt="square">
-      </div>
-      <div class="skills">
-        <h2 class="about_subtitle">Skills</h2>
-        <ul>
-          <li><h3>Codekit</h3></li>
-          <li><h3>Codepen</h3></li>
-          <li><h3>GitHub</h3></li>
-          <li><h3>GitLab</h3></li>
-          <li><h3>Terminal</h3></li>
-        </ul>
-        <img class="about_icon" src="/icons/bulb.svg" alt="light bulb">
+      <div id="about_left">
+        <div class="languages">
+          <h2 class="about_subtitle">Languages</h2>
+          <ul>
+            <li><h3>JavaScript(ES5/ES6)</h3></li>
+            <li><h3>Ruby</h3></li>
+            <li><h3>HTML</h3></li>
+            <li><h3>CSS</h3></li>
+          </ul>
+          <img class="about_icon" src="/icons/2.svg" alt="hashtag">
+        </div>
+        <div class="frameworks">
+          <h2 class="about_subtitle">Frameworks</h2>
+          <ul>
+            <li><h3>Bootstrap</h3></li>
+            <li><h3>Ruby on rails</h3></li>
+            <li><h3>RSpec</h3></li>
+            <li><h3>Capibara</h3></li>
+            <li><h3>Selenium</h3></li>
+          </ul>
+          <img class="about_icon" src="/icons/square.svg" alt="square">
+        </div>
+        <div class="skills">
+          <h2 class="about_subtitle">Skills</h2>
+          <ul>
+            <li><h3>Codekit</h3></li>
+            <li><h3>Codepen</h3></li>
+            <li><h3>GitHub</h3></li>
+            <li><h3>GitLab</h3></li>
+            <li><h3>Terminal</h3></li>
+          </ul>
+          <img class="about_icon" src="/icons/light_bulb.svg" alt="light bulb">
+        </div>
+        <div id="about_dots">
+          <img id="dots3" src="icons/dots_&_square.svg" alt="dots">
+        </div>
```

Comment:
- [ ] For this section, the hashtag, square and light bulb seem to be broken, kindly fix it.
![port41](https://user-images.githubusercontent.com/55635977/130150642-c9fcf40e-10a0-4a66-bbc0-0565e3784dca.png)